concent_api/conductor/models.py

```diff
@@ -70,6 +72,27 @@ class ResultTransferRequest(Model):
     modified_at = DateTimeField(auto_now=True)
 
 
+class ImageBorders(Model):
+
+    minimum_border_edge = DecimalField(max_digits=10, decimal_places=9)
+    maximum_border_edge = DecimalField(max_digits=10, decimal_places=9)
+
+
+class ImageResolution(Model):
+
+    x_dimension = IntegerField()
+    y_dimension = IntegerField()
+
+
+class BlenderCropScriptParameters(Model):
+
+    resolution = OneToOneField(ImageResolution, related_name='blender_crop_script_parameters')
+    samples = IntegerField()
+    use_compositing = BooleanField()
+    borders_y = OneToOneField(ImageBorders, related_name='blender_crop_script_parameters_as_borders_y')
+    borders_x = OneToOneField(ImageBorders, related_name='blender_crop_script_parameters_as_borders_x')
```

Comment:
This is not wrong but I'd personally just put the integers here instead of adding a relation:

 I.e.
``` python
resolution_x = IntegerField()
resolution_y = IntegerField()

borders_x_min = DecimalField(max_digits=10, decimal_places=9)
borders_x_max = DecimalField(max_digits=10, decimal_places=9)
borders_y_min = DecimalField(max_digits=10, decimal_places=9)
borders_y_max = DecimalField(max_digits=10, decimal_places=9)
```